private-templates-service/client/src/components/NavBar/NavBar.tsx

```diff
@@ -101,9 +101,9 @@ const NavBar = (props: NavBarProps) => {
       return (
         <Banner>
           <MobileBanner>
-            <StyledLogo aria-label={STRINGS.LOGO_DESCRIPTION} src={Logo} />
-            <Styledh1>{(props.templateID === "" && STRINGS.UNTITLEDCARD) || props.templateName}</Styledh1>
-            {props.templateID !== "" && <EditButton onClick={editName} iconProps={{ iconName: 'Edit' }} />}
+            <StyledLogo src={Logo} />
+            <Styledh1>{((props.templateID === "" || props.templateID === undefined) && STRINGS.UNTITLEDCARD) || props.templateName}</Styledh1>
```

Comment:
Correct me if I'm wrong but when the REST call is in progress I believe the template name becomes blank, can we handle that case using `isFetching`?

Reply:
I will create another ticket to address this thx